tests/unit/dom.ts

```diff
@@ -237,5 +237,169 @@ registerSuite({
 			assert.strictEqual(parent.children.length, 0);
 			assert.isNull(node.parentNode);
 		}
-	}
+	},
+
+	'CSS manipulation': (function () {
+		function cssSuite(element: Element): {} {
+			let targetElement = <any> element;
+			return {
+				addClass: {
+					afterEach() {
+						targetElement.className = '';
+					},
+
+					'add single class'() {
+						assert.notInclude(targetElement.className, 'test');
+						dom.addClass(targetElement, 'test');
+						assert.include(targetElement.className, 'test');
+					},
+
+					'add multiple classes'() {
+						dom.addClass(targetElement, 'test1', 'test2');
+						assert.include(targetElement.className, 'test1');
+						assert.include(targetElement.className, 'test2');
+					},
+
+					'null node'() {
+						assert.doesNotThrow(function () {
+							dom.addClass(null, 'test');
+						});
+					},
+
+					'invalid class'() {
+						assert.throws(function () {
+							dom.addClass(targetElement, 'te est');
+						});
+					},
+
+					'null class'() {
+						assert.doesNotThrow(function () {
+							dom.addClass(targetElement);
+						});
+					},
+
+					'existing class'() {
+						dom.addClass(targetElement, 'test');
+						dom.addClass(targetElement, 'test');
+						assert.lengthOf(targetElement.className.match(/test/), 1);
+					}
+				},
+
+				containsClass: {
+					afterEach() {
+						targetElement.className = '';
+					},
+
+					'contains single class'() {
+						dom.addClass(targetElement, 'test');
+						assert.isTrue(dom.containsClass(element, 'test'));
+					},
+
+					'null node'() {
+						assert.doesNotThrow(function () {
+							dom.containsClass(null, 'test');
+						});
+					},
+
+					'invalid class'() {
+						assert.throws(function () {
+							dom.containsClass(targetElement, 'te est');
+						});
+					},
+
+					'null class'() {
+						assert.doesNotThrow(function () {
+							dom.containsClass(targetElement, null);
```

Comment:
This *should* throw, according to the spec...

I assume this test case isn't testing passing nothing because TS makes that impossible without suppressing a warning.  I'd vote to just remove this test case for this function then.